src/service/command/gyokuon.ts

```diff
@@ -0,0 +1,76 @@
+import type {
+  CommandMessage,
+  CommandResponder,
+  HelpInfo
+} from './command-message.js';
+import type { Snowflake } from '../../model/id.js';
+import type { VoiceConnectionFactory } from '../voice-connection.js';
+import type { VoiceRoomController } from './kaere.js';
+
+export type GyokuonAssetKey = 'GYOKUON';
+
+const SCHEMA = {
+  names: ['gyokuon'],
+  subCommands: {}
+};
```

Comment:
```suggestion
import type { Schema } from '../../model/command-schema.js';
import type { Snowflake } from '../../model/id.js';
import type { VoiceConnectionFactory } from '../voice-connection.js';
import type { VoiceRoomController } from './kaere.js';

export type GyokuonAssetKey = 'GYOKUON';

const SCHEMA = {
  names: ['gyokuon'],
  subCommands: {}
} as const satisfies Schema;
```
このままだと型情報が欠落してしまうので, こうしておきましょう